nautobot_ssot_infoblox/diffsync/models/nautobot.py

```diff
@@ -87,6 +90,39 @@ def create(cls, diffsync, ids, attrs):
             status=status,
             description=attrs.get("description", ""),
         )
+        if attrs.get("vlans") and len(attrs["vlans"].keys()) >= 1:
```

Comment:
I would say that the `len(attrs["vlans"].keys()) >= 1` is redundant... If you get something that is not empty from "vlans" it's enough

Reply:
Good point